datavec/datavec-arrow/src/main/java/org/datavec/arrow/table/DataVecArrowUtils.java

```diff
@@ -0,0 +1,199 @@
+
+
+    /**
+     * Convert an existing data vec {@link Schema}
+     * to an {@link  org.bytedeco.arrow.Schema }
+     * @param schema the input schema
+     * @return the arrow schema
+     */
+    public static org.bytedeco.arrow.Schema toArrowSchema(Schema schema) {
+        Field[] fields = new Field[schema.numColumns()];
+        FieldVector schemaVector = new FieldVector(fields);
+        for(int i = 0; i < schema.numColumns(); i++) {
+            switch(schema.getType(i)) {
+                case Double:
+                    fields[i] = new Field(schema.getName(i),float64());
+                    break;
+                case NDArray:
+                    fields[i] = new Field(schema.getName(i),binary());
+                    break;
+                case Bytes:
+                    fields[i] = new Field(schema.getName(i),binary());
+                    break;
+                case String:
+                    fields[i] = new Field(schema.getName(i),utf8());
+                    break;
+                case Integer:
+                    fields[i] = new Field(schema.getName(i),int32());
+                    break;
+                case Time:
+                    fields[i] = new Field(schema.getName(i),date32());
+                    break;
+                case Categorical:
+                    fields[i] = new Field(schema.getName(i),utf8());
+                    break;
+                case Float:
+                    fields[i] = new Field(schema.getName(i),float32());
+                    break;
+                case Long:
+                    fields[i] = new Field(schema.getName(i),int64());
+                    break;
+                case Boolean:
+                    fields[i] = new Field(schema.getName(i),_boolean());
+                    break;
+            }
+        }
+
+        return new org.bytedeco.arrow.Schema(schemaVector);
+    }
+
+
+    /**
+     * Convert a {@link org.bytedeco.arrow.Schema }
+     * to a datavec {@link Schema}
+     * @param schema the input schema
+     * @return the {@link Schema}
+     */
+    public static Schema toDataVecSchema(org.bytedeco.arrow.Schema schema) {
+        Schema.Builder schemaBuilder = new Builder();
+        for(int i = 0; i < schema.num_fields(); i++) {
+            Field field = schema.field(i);
+            DataType dataType = field.type();
+            if(dataType.equals(arrow._boolean())) {
+                schemaBuilder.addColumnBoolean(field.name());
+            }
+            else if(dataType.equals(arrow.uint8())) {
+                schemaBuilder.addColumnInteger(field.name());
+            }
+            else if(dataType.equals(arrow.uint16())) {
+                schemaBuilder.addColumnInteger(field.name());
+            }
+            else if(dataType.equals(arrow.uint32())) {
+                schemaBuilder.addColumnInteger(field.name());
+            }
+            else if(dataType.equals(arrow.uint64())) {
+                schemaBuilder.addColumnInteger(field.name());
```

Comment:
uint32/64 should be mapped to Long to avoid underflow

Reply:
Done.